src/main/java/org/prebid/server/metric/RequestsMetrics.java

```diff
@@ -0,0 +1,18 @@
+package org.prebid.server.metric;
+
+import com.codahale.metrics.MetricRegistry;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+public class RequestsMetrics extends UpdatableMetrics {
```

Comment:
We don't need this class, use `org.prebid.server.metric.RequestMetrics` instead.